holodeck/agents.py

```diff
@@ -18,33 +19,59 @@ class ControlSchemes(object):
         UAV_TORQUES (int): Default UAV control scheme. Takes torques for roll, pitch, and yaw, as well as thrust.
         UAV_ROLL_PITCH_YAW_RATE_ALT (int): Control scheme for UAV. Takes roll, pitch, yaw rate, and altitude targets.
     """
+    # UAV Control Schemes
     ANDROID_TORQUES = 0
 
-    CONTINUOUS_SPHERE_DEFAULT = 0
-
-    DISCRETE_SPHERE_DEFAULT = 0
+    # Sphere Agent Control Schemes
+    SPHERE_DISCRETE = 0
+    SPHERE_CONTINUOUS = 1
 
+    # Nav Agent Control Schemes
     NAV_TARGET_LOCATION = 0
 
+    # UAV Control Schemes
     UAV_TORQUES = 0
     UAV_ROLL_PITCH_YAW_RATE_ALT = 1
 
 
+class AgentDefinition:
+    """A class for declaring what agents are expected or should be spawned in a particular holodeck Environment
+
+    Args:
+        agent_name (str): The name of the agent to control.
+        agent_type (str or type): The type of HolodeckAgent to control, string or class reference.
+        sensors (list of (SensorDefinition or type)): A list of HolodeckSensors to read from this agent.
```

Comment:
Try "sensors (list of SensorDefinition or class type (if no duplicate sensors))". Or something like it.